test/edu/umass/cs/gnsclient/client/integrationtests/ServerConnectTest.java

```diff
@@ -234,20 +213,20 @@ public static void setUpBeforeClass() throws Exception {
 	String gpConfFile = System.getProperty(DefaultProps.GIGAPAXOS_CONFIG.key);
 	String logFile = System.getProperty(DefaultProps.LOGGING_PROPERTIES.key);
 
-	ArrayList<String> output = RunServer.command("cat " + logFile + " | grep \"java.util.logging.FileHandler.pattern\" | sed 's/java.util.logging.FileHandler.pattern = //g'", ".", false);
+	ArrayList<String> output = RunCommand.command("cat " + logFile + " | grep \"java.util.logging.FileHandler.pattern\" | sed 's/java.util.logging.FileHandler.pattern = //g'", ".", false);
 	String logFiles = output.get(0) + "*";
 
 	System.out.println("Waiting for servers to be ready...");
-	output = RunServer.command("cat " + gpConfFile + " | grep \"reconfigurator\\.\" | wc -l ", ".", false);
+	output = RunCommand.command("cat " + gpConfFile + " | grep \"reconfigurator\\.\" | wc -l ", ".", false);
 	int numRC = Integer.parseInt(output.get(0));
-	output = RunServer.command("cat " + gpConfFile + " | grep \"active\\.\" | wc -l ", ".", false);
+	output = RunCommand.command("cat " + gpConfFile + " | grep \"active\\.\" | wc -l ", ".", false);
 	int numAR = Integer.parseInt(output.get(0));
 	int numServers = numRC + numAR;
-	output = RunServer.command("cat " + logFiles + " | grep \"server ready\" | wc -l ", ".", false);
+	output = RunCommand.command("cat " + logFiles + " | grep \"server ready\" | wc -l ", ".", false);
```

Comment:
This [may fail](http://unix.stackexchange.com/questions/19907/what-makes-grep-consider-a-file-to-be-binary) if the log file contains a NUL character. Turning on FINEST logging for NIO will put NUL characters in log files. Using `grep -a` will avoid this. (See `ServerIntegrationTest.java`)